Documentation/Advanced-Concepts/Serialization.html

```diff
@@ -110,7 +110,7 @@ <h2 id="method-1-adding-3-serialization-routines-to-your-type">Method 1: Adding
 <p> Unless you implement all three serialization routines, you should mark your type with the <code>Serializable</code> attribute so that the missing methods will be generated for you.</p>
 <h3 id="copier">Copier</h3>
 <p>Copier methods are flagged with the <code>Orleans.CopierMethod</code> attribute:</p>
-<pre><code class="lang-csharp">[CopierMethod]}
+<pre><code class="lang-csharp">[CopierMethod]
```

Comment:
the HTML files are auto-generated by a tool. You should fix the md file instead, which is the source of the HTML